.ci/docker/ci_commit_pins/nccl-legacy.txt

```diff
@@ -0,0 +1 @@
+ab2b89c4c339bd7f816fbc114a4b05d386b66290
```

Comment:
Q: Does it need to be a hash, or could it be a tag, say `v2.25.1-1`?

Reply:
HI @malfet yes tag would work as well.